src/elf/reloc.rs

```diff
@@ -191,7 +189,9 @@ macro_rules! elf_rela_std_impl { ($size:ident, $isize:ty) => {
                 let count = size / SIZEOF_RELA;
                 let mut relocs = vec![Rela::default(); count];
                 fd.seek(Start(offset as u64))?;
-                fd.read(relocs.as_mut_bytes())?;
+                unsafe {
+                    fd.read(plain::as_mut_bytes(&mut *relocs))?;
```

Comment:
A bit overestimated, yeah. That's pure aesthetic choice on my part.

The deal with `as_mut_bytes()` is that reading structure padding bytes may be undefined behavior in some cases. It's safe if the structure has no implicit padding, or if you only write without reading, but trying to encode that into the type system is too much trouble.